src/views/login-form.js

```diff
@@ -0,0 +1,23 @@
+import User from '../models/user';
+
+export default Backbone.View.extend({
+
+  events: {
+    submit(ev) {
```

Comment:
One thing to note is the `submit` event fires ONLY on `form` elements. Since your `.login-form` is a `div`, the `submit` function sill never fire.